widget/entry.go

```diff
@@ -1296,7 +1296,8 @@ func (r *entryContentRenderer) Objects() []fyne.CanvasObject {
 	defer r.content.entry.propertyLock.RUnlock()
 	// Objects are generated dynamically force selection rectangles to appear underneath the text
 	if r.content.entry.selecting {
-		return append(r.selection, r.objects...)
+		objs := append([]fyne.CanvasObject(nil), r.selection...)
```

Comment:
You should be able to do `[]fyne.CanvasObject{}` instead of `[]fyne.CanvasObject(nil)`.

Reply:
Yes, that's right, but I think `[]fyne.CanvasObject(nil)` is more performant because it does not allocate any memory at first time (`append` will create a new slice either with []fyne.CanvasObject{} or []fyne.CanvasObject(nil), however in the first one you have already allocated space that won't be useful)

Reply:
resolved in favor of the last change using `make`.